external-docs/docs/guides/configuration/qube-config.md

```diff
@@ -115,66 +120,588 @@ To reuse or extend existing dimensions, attributes, units, or measures, provide
+
+Measure and unit columns share some fields in that they can be populated directly in the column's definition with the `values` field, allowing a dictionary to be input that defines the details of the unit/measure. Existing column definitions can also be reused to quickly define measures and units, or serve as a "base" for a new column to be created on.
+
+TODO: Find out if cell_uri_template in measures/units has any distinctions compared to when it is used as a field in other column definitions, to avoid overlap/writing stuff about this field that is explained elsewhere in the guide.
+
+### Measures (for the guide)
+
+The *measure* column defines the phenomenon that is being observed, what is being quantified in this observation. In this way it is similar to a dimension. At least one measure is required to be defined in a data cube for it to be valid.
+
+### Measures Configuration
+
+A measure can be configured in two different ways: Either by attaching it inside an observation column's definition, or in a dedicated measures column of its own. When attaching a measure to an observation column, the measure can be defined after specifying that the type of the column is `observations`. The measure is entered as a dictionary attached to the observation column definition, like shown in the example below:
+
+### Measure configuration example in an Observation column field:
+
+This observation column definition is given a measure by creating a dictionary where the keys are the fields of the measure component, and the values are the field contents. This measure has the label "Median commute time" and the description of "The median amount of time taken to commute, in minutes".
+
+```json
+"columns": {
+  "Median commute time / mins": {
+    "type": "observations",
+    "measure": {
+      "label": "Median commute time",
+      "description": "The median amount of time taken to commute, in minutes"
+    }
+  }
+},
+```
+
+When creating a new measure column definition by specifying the `type` as "measures", the field details are entered into a `values` dictionary. Note this is different from how the measure details are given when giving the measure in an observation column.
+
+The following will describe the possible fields that can be entered into the `values` dictionary when defining a new measure, providing some examples of those fields being used to define a column.
+
+If basic measures are wanted, the `value` field can simply be set to True (as shown in the example below) which will indicate to csvcubed to create a measures column from this definition.
+
+TODO: Should we include a larger example with more columns and maybe some metadata when starting out? Then narrow down focus of examples later.
```

Comment:
Yes, I think we'll eventually want to break this document out into separate component types (e.g. a 'dimensions' configuration doc, a `measure` configuration document, etc.). When we do this, we'll want to assume that the user needs everything spelling out from the ground up (with links to where they can fill those gaps in); this means we'll want a fully-fledged JSON example at the top and we can then just do snippets after that.